pegasus/helpers/hoc_helpers.rb

```diff
@@ -151,6 +157,22 @@ def launch_tutorial(tutorial, params={})
       referer: request.referer_site_with_port,
       tutorial: tutorial[:code],
       company: params[:company],
+      source: ACTIVITY_TRACKING_SOURCES[:HOC],
+      started_at: DateTime.now,
+      started_ip: request.ip,
+    )
+  end
+
+  dont_cache
+  redirect tutorial[:url], 302
+end
+
+def launch_tutorial_learn(tutorial)
```

Comment:
This method looks awfully similar to `launch_tutorial`. Is there any reason the two shouldn't be combined, with an optional parameter `source`?